mycli/main.py

```diff
@@ -111,8 +112,11 @@ def __init__(self, sqlexecute=None, prompt=None,
             self.cnf_files = [defaults_file]
 
         # Load config.
-        config_files = ([self.default_config_file] + self.system_config_files +
-                        [self.user_config_file])
+        config_files = ([self.default_config_file] + self.system_config_files)
```

Comment:
Hmm I'm not sure; self.user_config_file is also used for writing the default configuration. I'm not sure why they are class variables and not just variables in __init__ it might be even better if user_config_file get it's default setting from click.